UndertaleModCli/Program.cs

```diff
@@ -341,8 +341,14 @@ private static int Dump(DumpOptions options)
             return EXIT_FAILURE;
         }
 
+        if (program.Data.IsYYC())
+        {
+            Console.WriteLine("Game is YYC, there is no code to dump. Exiting.");
+            return EXIT_SUCCESS;
+        }
+        
         // If user provided code to dump, dump code
-        if ((options.Code?.Length > 0) && (program.Data.Code.Count > 0))
+        if ((options.Code?.Length > 0) && (program.Data.Code is not null) && (program.Data.Code.Count > 0))
```

Comment:
Can't it be simplified as:
```suggestion
        if ((options.Code?.Length > 0) && (program.Data.Code?.Count > 0))
```

Reply:
It can, that's a leftover from when i wanted to try something else and forgot to remove it after.